inc/functions.php

```diff
@@ -0,0 +1,78 @@
+<?php
+/**
+ * Don't load directly.
+ *
+ * @package wp-google-login
+ */
+
+if ( ! defined( 'ABSPATH' ) ) {
+	die( '-1' );
+}
+
+/**
+ * Get Client id.
+ *
+ * @return mixed|string
+ */
+function wp_google_login_get_client_id() {
+	$options   = get_option( 'wp_google_login_settings' );
+	$client_id = '';
+	if ( defined( 'WP_GOOGLE_LOGIN_CLIENT_ID' ) ) {
+		$client_id = WP_GOOGLE_LOGIN_CLIENT_ID;
+	} else {
+		$client_id = ! empty( $options['client_id'] ) ? $options['client_id'] : '';
```

Comment:
I do not think that extra bracket has any added context here. Those are used in ternary operators only for readability. It would be logical to use those for i.e `( $foo == $bar )`